orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -236,8 +238,12 @@ def backward(ctx, *grad_outputs):
         # Get the module that flattens the output from the original module into a tuple
         self._flattened_output_module = \
             _ortmodule_output_transformation.get_flattened_output_module(self._original_module)
-        sig = signature(self._original_module.forward)
-        self._original_module_input_names = sig.parameters.keys()
+        self._original_module_parameters = signature(self._original_module.forward).parameters.values()
+        # TODO: remove after PyTorch ONNX exporter supports VAR_KEYWORD parameters.
+        for input_parameter in self._original_module_parameters:
+            if input_parameter.kind == inspect.Parameter.VAR_KEYWORD:
+                warnings.warn("The model's forward method has **kwargs parameter which is not supported.")
```

Comment:
As this is related to input, could we move it to `parse_inputs_for_onnx_export()`?

Reply:
Done.

Reply:
I put the check back into constructor because I want it to error during ORTModule creation, not during `forward`.